test/commands/continue_breakpoint_test.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+module Byebug
+  #
+  # Tests for continue command
+  #
+  class ContinueBreakpointTest < TestCase
+    def program
+      strip_line_numbers <<-RUBY
+         1:  module Byebug
+         2:    #
+         3:    # Toy class to test continue_breakpoint command.
+         4:    #
+         5:    class #{example_class}
+         6:      def factor(num)
+         7:        i = 1
+         8:        num.times do |new_number|
+         9:          i *= new_number
+        10:          byebug
+        11:        end
+        12:      end
+        13:    end
+        14:    c = 5
+        15:
+        16:    result = #{example_class}.new.factor(c)
+        17:    byebug
```

Comment:
This is an equally valid test but I would prefer to remove this line and instead add a "break 17" command before the `continue_breakpoint` command in the tests and assert that we land at line 17.

Reply:
I don't know if I undestand. Do you want remove the breakpoint in line 17?
If yes, I must test if stop in another breakpoint

Reply:
Yes, normally one drops a single `byebug` call where you want to start, and then add breakpoints from the command line instead of putting `byebug` calls everywhere. So I would prefer

```ruby
enter "break 17", "skip_breakpoint"
```

and then remove the second `byebug` breakpoint and let it stop at the new 17th line (the one with `"Result is: " + result.to_s`).

Reply:
Understood now :D

Changed